department-of-veterans-affairs/vets-website
# Wire no form page to feature toggle

# THIS IS A NEW FORM NOT IN PRODUCTION

## Summary
Add NoFormPage to wire up feature toggle on Income and Asset Statement Form (21P-0969)

## Issue
[Generate new vets-website application#81941](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81941)

### Related Issues
[Add feature toggle #81935](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81935)

### Epic
[Digitize Income and Asset Statement Form 21P-0969](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/80925)

### Screenshots
| Introduction Page | No Form Page |
| ---------- | --------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_introduction (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/963e80a3-a952-4cd8-b86a-c99b6497d86a)|![localhost_3001_income-and-asset-statement-form-21p-0969_introduction](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/2471bbf3-04d8-42a0-bfac-609cafe166ad)|


## How to run in local environment
1. Check out this branch locally in 'vets-website'
2.  Checkout this [branch](https://github.com/department-of-veterans-affairs/content-build/pull/2075) locally in 'content-build'
3. Run 'vets-api' in your local environment
4. Run the 'vets-website' app with `yarn watch --env entry=static-pages,auth,login-page,profile,dashboard,income-and-asset-statement`

## How to toggle feature flag in local environment
1. Go to [income_and_assets_form_enabled](http://localhost:3000/flipper/features/income_and_assets_form_enabled)

## How to verify
1. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser
2. Verify the 'Introduction' page is loaded when income_and_assets_form_enabled is enabled
3. Verify the 'No Form' page is loaded when income_and_assets_form_enabled is disabled

## Testing

## What areas of the site does it impact?
Income and Asset Statement Application

### Quality Assurance & Testing

### Error Handling

### Authentication
